jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
We don't take advantage of the recursors consul feature, we instead let Windows failover to the secondary DNS when Consul responds with a negative response. Does this mean that this wouldn't be a problem for us?

Also as a sidenote it looks like a configurable DNS timeout is on master: https://github.com/hashicorp/consul/pull/2322